config/locales/en.yml

```diff
@@ -377,9 +378,9 @@ en:
   profile:
     index:
       full_name: Full name
-      address: Address
-      dob: Date of Birth
-      ssn: Social Security Number (SSN)
+      address: Mailing address
```

Comment:
love that we're using I18n here. The locales file looks like it is getting pretty long. Maybe we should open an issue for splitting it up into multiple files? http://guides.rubyonrails.org/i18n.html#organization-of-locale-files